lib/src/buttons/help_button.dart

```diff
@@ -1,9 +1,9 @@
-import 'package:flutter/foundation.dart';
-import 'package:flutter/material.dart';
 import 'package:flutter/cupertino.dart'
     show CupertinoColors, CupertinoDynamicColor, CupertinoIcons;
-import '../../macos_ui.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
```

Comment:
Material and cupertino shouldn't be imported since `package:macos_ui/macos_ui.dart` already export the necessary widgets from these.